src/patterns/data/default/entity_attribute.tsv

```diff
@@ -4341,3 +4341,9 @@ OBA:2045294	asparaginase activity	GO:0006530	asparagine catabolic process	PATO:0
 OBA:2045295	brain connectivity attribute	UBERON:0000955	brain	PATO:0000141	structure			https://orcid.org/0000-0001-8314-2140
 OBA:2045296	defense response to other organism process attribute	GO:0098542	defense response to other organism	PATO:0001236	process quality			https://orcid.org/0000-0001-8314-2140
 OBA:2045328	DNA methylation level	GO:0006306	DNA methylation	PATO:0002062	physical quality of a process 			https://orcid.org/0000-0001-8314-2140
+OBA:2045372	gestational blood glucose level	UBERON:0009097	gravid organism	OBA:VT0000188	blood glucose amount			https://orcid.org/0000-0001-8314-2140
```

Comment:
This term exists [OBA:2045372](https://www.ebi.ac.uk/ols4/ontologies/oba/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FOBA_2045372) as `'gestational blood glucose trait'.` I see you have removed it from the EQL.tsv and created it here.

Reply:
Yes, I think this pattern is a more correct fit.